torchaudio/backend/utils.py

```diff
@@ -29,12 +33,13 @@ def list_audio_backends() -> List[str]:
     return backends
 
 
-def set_audio_backend(backend: Optional[str]) -> None:
+def set_audio_backend(backend: Optional[str]):
```

Comment:
nit: why are you removing the typing?

Reply:
This typing is redundant and messed up the documentation rendering.